.github/workflows/main.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
       - name: Run grit patterns test
         run: |
           if [ -n "${{ secrets.API_CLIENT_ID }}" ] && [ -n "${{ secrets.API_CLIENT_SECRET }}" ]; then
-              grit patterns test
+              grit patterns test --exclude ai
```

Comment:
Surely you can just remove the if/else if doing this?

Reply:
I plan to restore this very soon.